source/php/Helper/React.php

```diff
@@ -4,20 +4,29 @@
 
 class React
 {
-    public static function enqueue($version = false)
+    public static function enqueue()
     {
+
+        // Un-enqueue if React files are enqueued elsewhere
+        if (wp_script_is('react') || wp_script_is('react', 'registered')) {
+            wp_deregister_script('react');      
+            wp_dequeue_script('react');           
+        }
+        if (wp_script_is('react-dom') || wp_script_is('react-dom', 'registered')) {
+            wp_deregister_script('react-dom'); 
+            wp_dequeue_script('react-dom');   
+        }
+
         // Use minified libraries if SCRIPT_DEBUG is turned off
         $suffix = (defined('DEV_MODE') && DEV_MODE) ? 'development' : 'production.min';
-
-        $version = (is_string($version) && !empty($version)) ? $version : '16.4.2';
+        $version = '17.0.1';
 
         //Enqueue react
-        if (!wp_script_is('react')) {
-            wp_enqueue_script('react', 'https://unpkg.com/react@' . $version . '/umd/react.' . $suffix . '.js', array(), null);
-        }
+        wp_enqueue_script('react', EVENTMANAGERINTEGRATION_URL.'/source/js/front/react/react.' . $suffix  . '.js', array(), null);
+        wp_enqueue_script('react-dom', EVENTMANAGERINTEGRATION_URL.'/source/js/front/react/react-dom.' . $suffix  . '.js', array(), null);
+        //wp_enqueue_script('react', 'https://unpkg.com/react@' . $version . '/umd/react.' . $suffix . '.js', array(), null);
+        //wp_enqueue_script('react-dom', 'https://unpkg.com/react-dom@' . $version . '/umd/react-dom.' . $suffix . '.js', array(), null);
```

Comment:
if this is no longer necessary it should be removed

Reply:
and done